### PR TITLE
Gridview aligned at home page

### LINKED ISSUE #391: Improper grid view at home page
image
Third row first two columns are empty

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #391  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkboxe write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

GIF for the change
Peek 2019-03-29 23-30

Thanks!

### CODE REVIEW

**Comment by divyabaid16** [comment]
@publiclab/reviewers can you please review this?

File: app/assets/stylesheets/maps.css.scss [hunk 1]
```diff
@@ -33,10 +33,20 @@ html, body {
...
 .map-list .map h3 {
   overflow:hidden;
 }
 
+.map-list .map {
+  clear:none;
+  display: inline-block
+}
+
+.map-list div:nth-child(4n+1){
```
Comment (jywarren) [inline app/assets/stylesheets/maps.css.scss]: 😮😮😮😮😮 🎉 

This is super! 

Would it be possible to apply this to these styles as well? 

https://github.com/publiclab/plots2/blob/master/app/views/grids/_thumbnail.html.erb

**Comment by jywarren** [comment]
Thank you!!!!!!

**Comment by IshaGupta18** [comment]
Looks lovely!

File: app/assets/stylesheets/maps.css.scss (hunk 1 above)
Comment (divyabaid16) [reply app/assets/stylesheets/maps.css.scss]: Yes, I will look into them!!!

**Pull request merged**